tesseract/tesseract_planning/tesseract_motion_planners/test/ompl_trajopt_planner_test.cpp

```diff
@@ -119,32 +118,25 @@ static void addBox(tesseract_environment::Environment& env)
   env.addLink(link_1, joint_1);
 }
 
-template <typename PlannerType>
-class OMPLTrajOptTestFixture : public ::testing::Test
+void testHybridPlanner(tesseract_motion_planners::OMPLPlannerFreespaceConfig::Ptr ompl_config,
+                       tesseract_motion_planners::TrajOptPlannerFreespaceConfig::Ptr trajopt_config)
 {
-public:
-  using ::testing::Test::Test;
-  tesseract_motion_planners::OMPLTrajOptFreespacePlanner<PlannerType> planner;
-};
-
-using Implementations = ::testing::Types<ompl::geometric::SBL,
-                                         ompl::geometric::PRM,
-                                         ompl::geometric::PRMstar,
-                                         ompl::geometric::LazyPRMstar,
-                                         ompl::geometric::EST,
-                                         ompl::geometric::LBKPIECE1,
-                                         ompl::geometric::BKPIECE1,
-                                         ompl::geometric::KPIECE1,
-                                         // ompl::geometric::RRT,
-                                         // ompl::geometric::RRTstar,
-                                         // ompl::geometric::SPARS,
-                                         // ompl::geometric::TRRT,
-                                         ompl::geometric::RRTConnect>;
-
-TYPED_TEST_CASE(OMPLTrajOptTestFixture, Implementations);
-
-TYPED_TEST(OMPLTrajOptTestFixture, OMPLTrajOptFreespacePlannerUnit)  // NOLINT
```

Comment:
You could probably retain the template test structure here as well and just replace the implementations with the new planner configurator classes

Reply:
Switched